drivers/npmx/npmx_shell.c

```diff
@@ -962,7 +962,6 @@ static int cmd_buck_mode_set(const struct shell *shell, size_t argc, char **argv
 
 	if (argc < 3) {
 		shell_error(shell, "Error: missing buck mode.");
-		return 0;
```

Comment:
Why it is removed?

Reply:
Accidentally removed while rebasing, fixed.